frontend/catalyst/compiler.py

```diff
@@ -213,6 +213,7 @@ class MLIRToLLVMDialect(PassPipeline):
         "--convert-index-to-llvm",
         "--convert-gradient-to-llvm",
         "--convert-quantum-to-llvm",
+        "--wrapper-for-wrapper",
```

Comment:
I would suggest something with more meaning like
```suggestion
        "--normalize-ciface-signature",
```

Reply:
We are not normalizing the ciface signature, but adding a new pyface function. What do you think about: `--emit-catalyst-py-interface`?

Reply:
Sounds great 👍

Reply:
https://github.com/PennyLaneAI/catalyst/pull/106/commits/b8a8d001ab55027963dc9791957173abe32f8aa6